include/ttmlir/Conversion/Passes.td

```diff
@@ -21,6 +21,12 @@ def ConvertTosaToTTIR : Pass<"convert-tosa-to-ttir", "::mlir::ModuleOp"> {
   let dependentDialects = ["mlir::tt::ttir::TTIRDialect"];
 }
 
+def ConvertTTIRToTTIR: Pass<"convert-ttir-to-ttir", "::mlir::ModuleOp"> {
```

Comment:
Makes sense, @sdjordjevicTT suggested TTIRToTTIRDecomposition as a name

Reply:
> Sorry, I said TTIRToTTIR, but maybe we should have a more descriptive name. We'll want to run a different TTIRToTTIR dialect conversion for the TTMetal backend pipeline.

Should we have a TTIRToTTIR pipeline for common conversions which are agnostic to the backend? A good example is fusion pass in order to reconstruct ReLu operation. Ideally, only passes specific to TTNN or TTMetal should be included in the TTIRToTTNN and TTIRToTTMetal pipelines respectively, otherwise we will have a lot of duplication.

Reply:
@uazizTT that's a good idea, I think for now we can probably just assume that TTMetal backend will also first run `TTIRToTTIRDecomposition` before doing further decomposition.